.gitignore

```diff
@@ -9,3 +9,4 @@ scripts/CNAME
 .DS_Store
 *.swp
 npm-debug.log
+.nvmrc
```

Comment:
same here, if you changed this in one PR, don't submit the same change in another PR